helm/values.yaml

```diff
@@ -1,5 +1,6 @@
 environment: development
 image: mine-support
+auth: ZGVmcmE6JGFwcjEkY1drUVhUTU4kWUh5OERFT1pvMEM1MGF4WkpXREtuMAo=
```

Comment:
Should we be checking secrets in, even if they're temporary and encrypted?

Reply:
The secret is a placeholder used for local development, it isn't used in production and is overridden upon deployment.

Removing the 'secret' would add development overhead as prior to running locally a developer would need to generate an htpasswd file, and create a Kubernetes Secret to be used by the authentication.

I'll add details to the README to clarify it is a placeholder and is overridden on deployment.

Reply:
The approach was discussed in a Slack conversation.
The decision was to only apply basic-auth if an auth value is set on deployment. This keeps the auth value secret in the release pipeline, and enables local development without too much set up.